src/test/java/step1/SetTest.java

```diff
@@ -0,0 +1,45 @@
+package step1;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 크기를 확인")
+    void checkSize() {
+        assertThat(numbers).hasSize(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    @DisplayName("1, 2, 3의 값이 존재하는지 확인")
+    void checkValues(int value) {
+        assertThat(numbers.contains(value)).isTrue();
```

Comment:
```suggestion
        assertThat(numbers).contains(value);
```

collection의 경우 위와 같이 구현도 가능함